handlers/get_questions.go

```diff
@@ -0,0 +1,122 @@
+package handlers
+
+import (
+	"strings"
+	"github.com/DTS-STN/benefit-service/src/questions"
+	"github.com/DTS-STN/benefit-service/src/openfisca"
+	"github.com/DTS-STN/benefit-service/bindings"
+	"github.com/DTS-STN/benefit-service/renderings"
+	"github.com/labstack/echo/v4"
+	"net/http"
+	"time"
+)
+
+type Persons struct {
+	Person interface{} `json:"person"`
+}
+
+// Method for finding if array contains string
+func contains(s []string, str string) bool {
+	for _, v := range s {
+		if v == str {
+			return true
+		}
+	}
+
+	return false
+}
+
+// GetQuestions ...
+// @Summary Returns question dependencies for requested benefits
+// @Description Returns question dependencies for requested benefits
+// @ID GetBenefitQuestions
+// @Accept  json
+// @Produce json
+// @Param body body bindings.BenefitQuestionsRequest true "Benefits List"
+// @Success 200 {object} renderings.BenefitQuestionsResponse
+// @Failure 400 {object} renderings.BenefitServiceError
+// @Failure 404 {object} renderings.BenefitServiceError
+// @Failure 500 {object} renderings.BenefitServiceError
+// @Router /getquestions [get]
+func (h *Handler) GetQuestions(c echo.Context) error {
+	// Initialize request and response objects
+	getQuestionsRequest := new(bindings.BenefitQuestionsRequest)
+	getQuestionsResponse := new(renderings.BenefitQuestionsResponse)
+
+	// Bind incoming data to request object
+	if err := c.Bind(getQuestionsRequest); err != nil {
+		c.Logger().Error(err)
+		return c.JSON(http.StatusBadRequest, err)
+	}
+
+	// Copy request field values to response
+	getQuestionsResponse.RequestDate = getQuestionsRequest.RequestDate
+	getQuestionsResponse.LifeJourneys = getQuestionsRequest.LifeJourneys
+	getQuestionsResponse.BenefitList = getQuestionsRequest.BenefitList
```

Comment:
Why are these returned if they are not modified. The caller already has this information.
LifeJourneys is also unused